### DISCUSSION TITLE
Is it possible to show what items something (eg enchanted book) can be added to?

### DISCUSSION THREAD

**Discussion by The-Epic** [shedaniel/RoughlyEnoughItems#1172] [Q & A] [answered]
On (i think) JEI it shows what items an enchanted book can go on, i was wondering if this can be enabled or if there is a plugin for it. Looks like 
![image](https://user-images.githubusercontent.com/87966707/198147351-fa7567ff-fd68-4708-8a46-8632499dcd4b.png)

**Comment by shedaniel** [comment, marked as answer]
I don't think that is done by JEI.

**Comment by SpokenWig620933** [comment]
That is done by Quark if you didn't know by now.
